admin_guide/allocating_node_resources.adoc

```diff
@@ -0,0 +1,115 @@
+= Allocating Node Resources
```

Comment:
Title change to keep with gerund-style headings. Please LMK if this is an inaccurate title for any reason.